NightscoutServiceKit/RemoteCommands/NightscoutRemoteCommand.swift

```diff
@@ -1,36 +0,0 @@
-//
```

Comment:
With the refactor, now there are the following objects:

V1 (Loop now): RemoteNotification: An incoming push notification. 
V2 (remote 2.0): NSRemoteCommandPayload: Payload for a incoming push notification and also when making REST calls to NS for pending commands

Both V1 and V2 will use the same Actions which are above.